api/python/tests/integration/test_packages.py

```diff
@@ -532,7 +534,12 @@ def test_load_into_quilt(self):
 
         with patch('time.time', return_value=1234567891), \
              patch('quilt3.data_transfer.s3_transfer_config.max_request_concurrency', 1):
-            remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            stdout = io.StringIO()
+
+            # temporarily redirect stderr to capture warnings (usually errors)
+            with redirect_stderr(stdout):
+                remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            assert '100%|##' not in stdout.getvalue()
```

Comment:
this is a bit fragile. what if it gets to 99% and fails. check a bit more carefully for the absence of tqdm data.